app/models/metric/ci_mixin.rb

```diff
@@ -19,12 +19,17 @@ module Metric::CiMixin
     Metric::LongTermAverages::AVG_METHODS_WITHOUT_OVERHEAD.each do |vcol|
       virtual_column vcol, :type => :float, :uses => :vim_performance_operating_ranges
     end
+  end
 
-    supports :capture do
-      metrics_capture_klass = "#{self.class.module_parent.name}::MetricsCapture".safe_constantize
-      unless metrics_capture_klass&.method_defined?(:perf_collect_metrics)
-        _('This provider does not support metrics collection')
-      end
```

Comment:
NOTE since this `CiMixin` is included in base classes like `::VmOrTemplate` and `::Host` so every subclass would report that it supported capture, even if it didn't.  You had to check each instance in order to tell if the _class_ supported metrics.